micronaut-webapp/api/src/main/java/org/graalvm/demos/micronaut/service/api/v1/Todo.java

```diff
@@ -4,9 +4,10 @@
 
 import javax.validation.constraints.NotBlank;
 import java.util.Date;
+import java.util.Objects;
```

Comment:
Is this import unused?

Reply:
good catch, I'll fix that on the next PR